recipes-utils/exfatprogs/exfatprogs/run-ptest

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
+
+rm -f ptest.log || true
+rm -f test_create.log || true
+
+loopdev=$(losetup -f)
+
+mountdir="/mnt/exfatprogs_ptest"
+
+if [ ! -d "${mountdir}" ]; then
+  mkdir "${mountdir}"
+fi
+
+exfat_support=$(grep exfat /proc/filesystems >/dev/null ; echo $?)
+module_installed=0
+
+if [ "${exfat_support}" = "1" ]; then
+  insmod /lib/modules/$(uname -r)/kernel/fs/exfat/exfat.ko
+  if [ $? -ne 0 ]; then
+    echo "Error: This kernel is not support exfat filesystem."
+    exit 1
+  fi
+  module_installed=1
+fi
+
+cd tests
+./test_fsck.sh 2>&1 | tee ../ptest.log
+cd -
+output=$(tac ./ptest.log | head -n 1)
+
+passed=$(echo "$output" | awk '{print $2}')
+all=$(echo "$output" | awk '{print $4}')
+failed=$((all - passed))
+skipped=0
+
+ret_val=1
+
+if [ "$failed" -eq 0 ]; then
+  echo "fsck test: All $all tests passed"
+  ret_val=0
+else
+  echo "$failed of $all tests failed"
+fi
+
+if [ $ret_val = 1 ]; then
+  echo "Error: fsck test was failed."
+  skipped=1
+else
+  cd create_test
+  ./test_create.sh "${mountdir}" "${loopdev}" 2>&1 | tee ../test_create.log
+  cd -
+
+  create_test_result=$(grep "create test passed" ./test_create.log >/dev/null ; echo $?)
```

Comment:
"create test passed" -> "create passed test list" or  "creating list of passed test"

Reply:
I think the script name "test_create.sh" and the message "create test passed" are confusing.
For more clarity, the script name should be "test_file_creation.sh" and the message should be "file/directory creation test passed."

Reply:
@aliceinwire @hiraku-wfs I committed the updates that addressed all the review comments.

Reply:
I think "test_log_creation.sh" or "test_logfile_creation.sh" sounds better or we don't know what kind of file we are creating.

Reply:
@masami256 @aliceinwire Looking at the contents of test_file_creation.sh, it doesn't seem like this has anything to do with logs or log files, is that correct?

Reply:
@hiraku-wfs I see what you mean, that's right.

Reply:
Thanks!

Reply:
Okay. will keep the test_file_creation.sh as script name.